config/routes.rb

```diff
@@ -3,11 +3,10 @@
     root to: 'dashboards#index', as: :authenticated_root
     get 'dashboard', to: 'dashboards#index', as: 'dashboard'
     post 'search', to: 'dashboards#search', defaults: { format: :js }
-    resources :pregnancies, only: [ :edit, :update ] do
+    resources :pregnancies, only: [ :create, :edit, :update ] do
```

Comment:
Let's do this: I'll handle this change in a separate pull request. For now, let's roll with what you've got.